src/pages/assets.tsx

```diff
@@ -0,0 +1,479 @@
+import ethDiamondColor from "@/public/assets/eth-diamond-rainbow.png"
+import ethDiamondGlyph from "@/public/assets/eth-diamond-glyph.png"
+import ethDiamondPurple from "@/public/assets/eth-diamond-purple.png"
+// import ethDiamondPurpleHero from "@/public/assets/eth-diamond-purple.png"
+import ethDiamondPurplePurple from "@/public/assets/eth-diamond-purple-purple.png"
+import ethDiamondPurpleWhite from "@/public/assets/eth-diamond-purple-white.jpg"
+import ethGlyphColored from "@/public/assets/eth-glyph-colored.png"
+import ethLandscapeBlack from "@/public/assets/ethereum-logo-landscape-black.png"
+import ethLandscapeBlackGray from "@/public/assets/ethereum-logo-landscape-black-gray.png"
+import ethLandscapePurple from "@/public/assets/ethereum-logo-landscape-purple.png"
+import ethLandscapePurplePurple from "@/public/assets/ethereum-logo-landscape-purple-purple.png"
+import ethLandscapePurpleWhite from "@/public/assets/ethereum-logo-landscape-purple-white.png"
+import ethPortraitBlack from "@/public/assets/ethereum-logo-portrait-black.png"
+import ethPortraitBlackGray from "@/public/assets/ethereum-logo-portrait-black-gray.png"
+import ethPortraitPurple from "@/public/assets/ethereum-logo-portrait-purple.png"
+import ethPortraitPurplePurple from "@/public/assets/ethereum-logo-portrait-purple-purple.png"
+import ethWordmarkBlack from "@/public/assets/ethereum-wordmark-black.png"
+import ethWordmarkBlackGray from "@/public/assets/ethereum-wordmark-black-gray.png"
+import ethWordmarkPurple from "@/public/assets/ethereum-wordmark-purple.png"
+import ethWordmarkPurplePurple from "@/public/assets/ethereum-wordmark-purple-purple.png"
+import ethWordmarkPurpleWhite from "@/public/assets/ethereum-wordmark-purple-white.png"
+import finance from "@/public/finance_transparent.png"
+import future from "@/public/future_transparent.png"
+import hackathon from "@/public/hackathon_transparent.png"
+import hero from "@/public/home/hero.png"
+import heroPanda from "@/public/home/hero-panda.png"
+import impact from "@/public/impact_transparent.png"
+import infrastructure from "@/public/infrastructure_transparent.png"
+import merge from "@/public/upgrades/merge.png"
+import mergePanda from "@/public/home/merge-panda.png"
+import newRings from "@/public/upgrades/newrings.png"
+import oldShip from "@/public/upgrades/oldship.png"
+import wallet from "@/public/wallet.png"
+import whatIsEthereum from "@/public/what-is-ethereum.png"
+// import efLogo from "@/public/ef-logo.png"
+// import efLogoWhite from "@/public/ef-logo-white.png"
+// import ethGifCat from "@/public/eth-gif-cat.png"
+// import ethGifChalk from "@/public/eth-gif-chalk.png"
+// import ethGifSun from "@/public/eth-gif-sun.png"
+// import ethGifWaves from "@/public/eth-gif-waves.png"
+// import ethPortraitPurpleWhite from "@/public/assets/ethereum-logo-portrait-purple-white.png"
+// import leslieTheRhino from "@/public/upgrades/upgrade_rhino.png"
```

Comment:
From what I can tell, we were importing these in the Gatsby repo, but not actually displaying them. Left them here commented out in the meantime, but we should probably either display these or remove them.

Reply:
Ok thats odd but yea, lets keep them just in case.